eclair-core/src/main/scala/fr/acinq/eclair/NodeParams.scala

```diff
@@ -244,7 +246,7 @@ object NodeParams {
       onChainFeeConf = OnChainFeeConf(
         feeTargets = feeTargets,
         feeEstimator = feeEstimator,
-        maxFeerateMismatch = config.getDouble("on-chain-fees.max-feerate-mismatch"),
+        maxFeerateMismatch = FeerateMismatch(config.getDouble("on-chain-fees.feerate-mismatch.ratio-low"), config.getDouble("on-chain-fees.feerate-mismatch.ratio-high")),
```

Comment:
```suggestion
        maxFeerateMismatch = FeerateTolerance(config.getDouble("on-chain-fees.feerate-mismatch.ratio-low"), config.getDouble("on-chain-fees.feerate-mismatch.ratio-high")),
```

Reply:
Fixed in https://github.com/ACINQ/eclair/pull/1473/commits/9037da2978266fb0de40bf13c272d0024dfedcea